e2e/workflow/installation.go

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Awesome, these look like Installation creation/deletion e2e tests which we also need outside of the context of db migrations so this is really two-for-one here :grinning: